packages/aws-cdk-lib/aws-stepfunctions/lib/util.ts

```diff
@@ -1,4 +1,37 @@
+import { IKey } from '../../aws-kms';
+import { Duration } from '../../core';
+
 export function noEmptyObject<A>(o: Record<string, A>): Record<string, A> | undefined {
   if (Object.keys(o).length === 0) { return undefined; }
   return o;
+}
+
+function isInValidKmsDataKeyReusePeriodSeconds(kmsDataKeyReusePeriodSeconds: Duration) {
```

Comment:
```suggestion
function isInvalidKmsDataKeyReusePeriodSeconds(kmsDataKeyReusePeriodSeconds: Duration) {
```

Assuming you mean "invalid" and not "in valid range".